molo/forms/models.py

```diff
@@ -192,6 +192,12 @@ class MoloFormPage(
         verbose_name='Is Contact Form',
         help_text='This will display the correct template for contact forms'
     )
+    save_article_object = BooleanField(
```

Comment:
@moh-moola why are we making this optional?

Reply:
Not all forms are linked to an article, `MoloForm`s are stand alone `Page` objects, we have not been storing any `Article Page` objects to `MoloFormSubmission` now it makes sense to add this functionality to store the `ArticlePage` for the reaction questions. This will then make it possible to limit result calculations per `MoloForm` or (`MoloForm`, `ArticlePage`)

Reply:
@moh-moola but why is a separate flag needed for this on the model? You're not using this flag to limit result calculations. You actually aren't checking anything at that point `return prev_step.success(self.slug, form.cleaned_data.get('article_page'))` (this will raise an error if the form doesn't have that field)

Instead I think you can leave it like it was. Always add the `article_page` field to the form and treat submissions differently if that is empty